src/utils/i18n.js

```diff
@@ -0,0 +1,17 @@
+import { I18n } from 'i18n-js';
+import * as Localization from 'expo-localization';
+
+import en from './locales/en';
+import fr from './locales/fr';
+
+const i18n = new I18n({
+  en,
+  fr,
+});
+
+i18n.defaultLocale = 'en';
```

Comment:
Maybe only fr should be configured for now, since the en translation will come later (#48)